plugins/woocommerce-admin/client/wp-admin-scripts/product-category-metabox/style.scss

```diff
@@ -0,0 +1,37 @@
+.product-add-category {
+	&__tree-control {
+		margin-top: 8px;
```

Comment:
Do we have access to any SCSS variables so that we don't have to hardcode values here?

Reply:
Yup added where possible.